sdk/samples/azidentity/manual-tests/webjobs/main.go

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Why is this under a `webjobs` directory?  This seems like a pretty generic sample for retrieving a token via MSI.

Reply:
I had originally thought the tests would branch off and become different at some point, but you're right. I'm changing it to managed_identity since that's all it does at the moment.